### PR TITLE
Feature/book card

### PR DESCRIPTION
**Pull request by sphoorthigaddam**
Adding Book Card feature.

### CODE REVIEW

**Review by mihirgupta0900 — CHANGES_REQUESTED** [review]
On line no. 27, there is a syntax error. It should be `image={props.thumbnail}` and not `image="{props.thumbnail}"`.
"" imply that it is a string, to use variables, just use `{variable_name}`.

Please correct it in other places too.

Also, deconstruct the props while receiving it.
For eg, instead of `BookCard(props)`, replace it with `BookCard({thumbnail, rating, book_title, author})`.
It is not a syntax error, but helps in code maintainability.

And for the styles, add a bit of left padding for the stars

File: src/components/BookCard.js [hunk 1]
```diff
@@ -15,22 +15,28 @@ const useStyles = makeStyles({
...
+	rating: {
+		paddingLeft: 15
+	},
+	typography: {
+		paddingLeft: 5
 	}
 });
 
-function BookCard(props) {
+function BookCard(thumbnail, rating, book_title, author) {
```
Comment (mihirgupta0900) [inline src/components/BookCard.js]: It needs to be `({thumbnail, rating, book_title, author})` and not `(thumbnail, rating, book_title, author)`.

What the first code means is you are taking the props and refactoring it.
```js
function BookCard(props) {
    const {thumbnail, rating, book_title, author} = props
}
```

```js
function BookCard({thumbnail, rating, book_title, author}) {
    
}
```

The above two code blocks mean the same thing. I hope it is clearer now? Please make these changes and create another commit.
Thank You

**Review by mihirgupta0900 — CHANGES_REQUESTED** [review]

File: src/components/BookCard.js (hunk 1 above)
Comment (sphoorthigaddam) [reply src/components/BookCard.js]: Yes this makes sense. I will change it right away.

Comment (sphoorthigaddam) [reply src/components/BookCard.js]: @mihirgupta0900 changes made. Let me know if it looks good.

**Review by mihirgupta0900 — APPROVED** [review]
